run_dir/static/js/proj_meta_compare.js

```diff
@@ -1,604 +1,616 @@
-    }
-    if(data.length == 0){
-        $('#proj_meta_plot').text('No data to show');
-    }
+        // List skipped samples
+        if(skipped_samples.length > 0){
+            var stat_string = '<strong>Warning:</strong> '+skipped_samples.length+' samples out of ' + (num_data + skipped_samples.length) + ' skipped:<br>';
+            $.each(proj_skipped, function(pid, count){
+            if(count > 0){
+                stat_string += '<code>'+pid+'</code>: '+count+' out of ' + Object.keys(project_data[pid]).length + ' samples skipped<br>';
+            }
+            });
+            stat_string += ' <br><button class="btn btn-outline-dark btn-sm" onClick="$(\'#skipped_samples_list\').slideToggle();">show / hide hidden sample names</button>'+
+            '<pre id="skipped_samples_list" style="display:none;">'+skipped_samples.join("\n")+'</pre>'
+            $('#skipped_status_tip').html(stat_string);
+        } else {
+            $('#skipped_status_tip').html('<strong>Good news:</strong> ' + num_data + ' samples plotted, none skipped!');
+        }
+        if(data.length == 0){
+            $('#proj_meta_plot').text('No data to show');
+        }
 
-    var ytitle = keys['y'][0]+': '+keys['y'][1];
-    var xtitle = keys['x'][0]+': '+keys['x'][1];
-    $('#proj_meta_plot').highcharts({
-        chart: {
-            type: 'scatter',
-            zoomType: 'xy',
-            height: 800,
-            plotBorderWidth: 1,
-            plotBackgroundColor: '#FAFAFA',
-            events: {
-                load: function(event) {
-                    calc_corr_score(this.series);
-                },
-                redraw: function(event) {
-                    calc_corr_score(this.series);
+        var ytitle = keys['y'][0]+': '+keys['y'][1];
+        var xtitle = keys['x'][0]+': '+keys['x'][1];
+        $('#proj_meta_plot').highcharts({
+            chart: {
+                type: 'scatter',
+                zoomType: 'xy',
+                height: 530,
+                plotBorderWidth: 1,
+                plotBackgroundColor: '#FAFAFA',
+                events: {
+                    load: function(event) {
+                        calc_corr_score(this.series);
+                    },
+                    redraw: function(event) {
+                        calc_corr_score(this.series);
+                    }
                 }
-            }
-        },
-        title: {
-            text: '<u>'+ytitle+'</u> compared to <u>'+xtitle+'</u>',
-            useHTML: true
-        },
-        credits: {
-            enabled: false
-        },
-        yAxis: {
-            title: {
-                text: ytitle
             },
-            type: xLogAxis,
-            lineWidth: 1,
-            gridLineWidth: 0
-        },
-        xAxis: {
             title: {
-                text: xtitle
+                text: '<span class="meta_plot_cats"><i>"'+ytitle+'"</i></span> compared to <span class="meta_plot_cats"><i>"'+xtitle+'"</i></span>',
```

Comment:
Would be nice if the projects were also displayed on top. I feel it's too small and to the side currently.

Reply:
You mean the legend? I moved it from the right side to the left, but I guess we could also put it on top 🤷🏻‍♀️